docs/documentation/maya/simshape.md

```diff
@@ -38,90 +37,124 @@ In order to add or remove any of those optional meshes, a set of menu items are
+ - **Compute Collisions** (Boolean, True): Flag to enable collisions correction in the deformer. If disabled, the deformer will ignore colliders when deforming the mesh.
+ - **Keep Orientation** (Boolean, True): Flag to preserve the initial orientation of the vertices relative to the collider when handling collisions. If disabled, the mesh will suffer no changes if the orientation of the collider varies.
+ - **Max Sliding Distance** (Float, 1.0): Maximum distance (in world units) the simulated vertex is allowed to slide relative to the collider.
+    - Has a range of \[0.0, 10.0\] [^1]
 
 #### Attraction Settings
-| Attribute             | Type | Value     | Range/Options  | Description           |
-| :------------         | :--- | :----     | :------------  | :----------           |
-| Attraction Remap Mode | Enum | Cube Root | <ul><li>Linear</li><li>Logarithmic</li><li>Square Root</li><li>Cube Root</li></ul> | Remap mode used to compute the definitive attraction values. |
+ - **Attraction Multiplier** (Float, 1.0): Maximum distance (in world units) the simulated vertex is allowed to slide relative to the collider.
+    - Has a range of \[0.0, 10.0\] [^1]
+ - **Attraction Remap Mode** (Enumerator, Cube Root): Remap mode used to compute the definitive attraction values. There are 4 different modes that folow different remap methods:
+    <ul><li>Linear</li><li>Logarithmic</li><li>Square Root</li><li>Cube Root</li></ul>
 
 #### Initialization Settings
-| Attribute               | Type | Value | Range/Options   | Description           |
-| :------------           | :--- | :---- | :------------   | :----------           |
-| Animatable Rest Mesh    | Bool | False | \[False, True\] | Flag that enables reading animated rest mesh data. |
-| Initialize to Anim Mesh | Bool | False | \[False, True\] | Flag to instantiate points at animated mesh instead of rest mesh on initialization. |
+ - **Animatable Rest Mesh** (Boolean, False): Flag that enables reading animated rest mesh data.
+ - **Initialize to Anim Mesh** (Boolean, True): Flag to instantiate points at animated mesh instead of rest mesh on initialization.
 
 #### Activation Remap
-| Attribute        | Type           | Value  | Range/Options  | Description           |
-| :------------    | :---           | :----  | :------------  | :----------           |
-| Activation Remap | Ramp Attribute | Linear || Curve to remap the activation values. |
-
-#### Additional Properties
-| Attribute     | Type  | Value | Range/Options  | Description                  |
-| :------------ | :---  | :---- | :------------  | :--------------------------- |
-| Attract Force | Float | 1.0   | \[0.0, 1.0\]   | Attibute to control the amount of influence of the animated mesh. The higher the value is, the more influence and the less dynamics will appear. This attribute is paintable. |
+ - **Activation Remap** (Ramp Attribute): Curve to remap the activation values.
+
+## Paintable Weights
```

Comment:
I notice in this section and elsewhere in the documentation that sometimes capital letters are used and sometimes lower case letters are used after colons. I think we should follow the same criteria to maintain consistency throughout the documentation.

Reply:
You are right! I believe lower case is more correct after semicolon.

Similar to using italics when referencing attributes, this is something we should do a full review of throughout the documentation. I will take note of this for future reference.

Reply:
Added capital letters when the colon introduces a complete sentence.